data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1097,6 +1097,11 @@ spec:
                   clusterOSImage:
                     description: ClusterOSImage is either a URL with `http(s)` or `file` scheme to override the default OS image for cluster nodes, or an existing Glance image name.
                     type: string
+                  clusterOSImageProperties:
+                    additionalProperties:
+                      type: string
+                    description: ClusterOSImageProperties is a list of properties to be added to the metadata of the uploaded Glance ClusterOSImage. in the install-config.yaml. hw_qemu_agent=yes would look like
```

Comment:
```suggestion
                    description: ClusterOSImageProperties is a list of properties to be added to the metadata of the uploaded Glance ClusterOSImage.
```

Reply:
Because this is not platform specific, I think that we should remove any reference to Glance or OpenStack in this. It should just say it will be added to the image metadata.